speakers/andraz-hribernik.md

```diff
@@ -1,7 +1,10 @@
 name: Andraz Hribernik
 ---
 Andraz Hribernik
-Andraz is Data Engineer at Cytora. He is interested in big data processing, best engineering practices and ML. More specifically, he is trying to build processing pipeline that combines various advanced algorithms to be as easy to use as sklearn library. Before joining Cytora Andraz earned MSc in Computer Science, did research internship at Stanford University and worked for two successful Slovenian startups (Zemanta, Celtra).
-
+- Andraz is Data Engineer at Cytora. He is interested in big data processing, engineering best practices and ML. More specifically, his aim is to build a processing pipeline that combines various advanced algorithms to be as easy to use as the sklearn library. Before joining Cytora, Andraz earned an MSc in Computer Science, did a research internship at Stanford University and worked for two successful Slovenian startups (Zemanta, Celtra).
```

Comment:
Similarly, if you want these to show up as bullets, you need to add a newline before and after the lines beginning with a hyphen.